vagrant_opm_user/bootstrap.sh

```diff
@@ -20,9 +20,9 @@ sudo apt-get update -y
 
 # OPM packages
 sudo apt-get install libopm-core1-bin -y
-sudo apt-get install libopm-autodiff1-bin -y
-sudo apt-get install libopm-polymer1-bin -y
+sudo apt-get install libopm-simulators1-bin -y
 sudo apt-get install libopm-upscaling1-bin -y
+sudo apt-get install openmpi-bin
```

Comment:
it doesn't depend on the mpiexec binary. it only depends on the libraries, which sits in mpi-default-dev (meta package pulling in libopenmpi-dev right now). likewise, there's a meta package mpi-default-bin which pulls in the default mpiexec. users can reconfigure their default mpi binary on the system and rebuild packages (e.g. use mpich2 instead) without having to change packaging or apt-get commands. it's all very theoretical but it's how debian is designed so i try to follow suite. if you think convenience trumps being correct i can have the opm-simulators-bin depend on mpi-default-bin in the future.